### PR TITLE
Fix links in docs

### PR DESCRIPTION
**Pull request by Manishearth**
CI keeps complaining

### CODE REVIEW

File: components/icu/src/lib.rs [hunk 1]
```diff
@@ -169,7 +169,7 @@ pub mod decimal {
     //! assert_eq!("2,000.50", fdf.format(&fixed_decimal).writeable_to_string());
     //! ```
     //!
-    //! [`FixedDecimal`]: fixed_decimal::FixedDecimal
+    //! [`FixedDecimal`]: decimal::FixedDecimal
```
Comment (sffc) [inline components/icu/src/lib.rs:172]: Question: How does this work?  `FixedDecimal` is in the `fixed_decimal` crate, not the `icu_decimal` crate.

Comment (Manishearth) [reply components/icu/src/lib.rs:172]: Ah for some reason I thought it was a reexport

File: components/icu/Cargo.toml [hunk 2]
```diff
@@ -63,11 +63,14 @@ version = "0.2"
 path = "../uniset"
 default-features = false
 
+[dependencies.fixed_decimal]
```
Comment (sffc) [inline components/icu/Cargo.toml:66]: Question: This isn't used in `icu` except in the one docs string. Does that one docs string compel us to add the direct dependency?

**Review by sffc — APPROVED** [review]

File: components/icu/Cargo.toml (hunk 2 above)
Comment (Manishearth) [reply components/icu/Cargo.toml:66]: Yes, unless we can find it as a reexport. But it's already being built indirectly so it's not a big deal.

**Pull request merged**